test/exercism/guest_test.rb

```diff
@@ -1,8 +1,51 @@
 require_relative '../test_helper'
+require_relative '../active_record_helper'
```

Comment:
I guess it's not so surprising since Guest is basically a NullObject version of User which is an ActiveRecord object.